demo/terraform/demo_stack/stack.tf

```diff
@@ -1,10 +1,10 @@
 module "stack" {
-  source = "github.com/wellcomecollection/storage-service.git//terraform/modules/stack?ref=caab5b2"
+  source = "github.com/wellcomecollection/storage-service.git//terraform/modules/stack?ref=52e10ed"
```

Comment:
Can we create a tag specifically for the demo?

Reply:
Good suggestion. I haven't yet because the demo is still in flux, but one to revisit later.